app/models/ability.rb

```diff
@@ -62,4 +66,10 @@ def custom_permissions
     def curation_concerns
       Hyrax.config.curation_concerns
     end
+
+    def download_file_permission
+      can :read, ActiveFedora::File do |file|
+        file.mime_type == 'image/tiff' ? false : true
```

Comment:
Why is this check specifically targeting `image/tiff`? What are we trying to prevent here? The download of source/master files?